doc/tutorial/argument_parser/solution6.cpp

```diff
@@ -78,10 +78,10 @@ void initialize_argument_parser(argument_parser & parser, cmd_arguments & args)
     parser.info.short_description = "Aggregate average Game of Thrones viewers by season.";
     parser.info.version = "1.0.0";
 
-    //![file_existence_validator]
+    //![path_existence_validator]
     parser.add_positional_option(args.file_path, "Please provide a tab separated data file.",
-                                 file_existance_validator() | file_ext_validator({"tsv"}) );
-    //![file_existence_validator]
+                                 path_existence_validator() | file_ext_validator({"tsv"}) );
```

Comment:
```suggestion
                                 path_existence_validator{} | file_ext_validator({"tsv"}) );
```
do not change the file_ext_validator to braces because somebody else is working on them